pallets/nfts/src/lib.rs

```diff
@@ -1345,6 +1457,55 @@ pub mod pallet {
 			Self::do_cancel_approval(maybe_check_origin, collection, item, delegate)
 		}
 
+		/// Cancel a collection approval.
+		///
+		/// Origin must be Signed.
+		///
+		/// Arguments:
+		/// - `collection`: The collection whose approval will be cancelled.
+		/// - `delegate`: The account that is going to lose their approval.
+		///
+		/// Emits `ApprovalCancelled` on success.
+		///
+		/// Weight: `O(1)`
+		#[pallet::call_index(41)]
```

Comment:
This can wait till the end because it might not be necessary

Reply:
Reason why I structure it like this because if someone read the code, when they visit `approve_transfer`, they can also find `approve_collection_transfer`. But I understand your point of bringing this to the bottom of the file.

Reply:
Not 100% sure whether this is required so lets leave this open until @evilrobot-01 reviews

Reply:
This is resolved by latest commit and comments